src/Components/MainApp/App.js

```diff
@@ -7,42 +7,35 @@ import ExperimentTwo from '../ExperimentTwo/ExperimentTwo.js';
 
 function App() {
   // Button States
-  const [headerState, setHeaderState]=useState(true);
```

Comment:
I don't think this state is used for anything anymore? Should be able to get rid of it.

Reply:
lol ignore me - this is already removed...